includes/class-wc-stripe-apple-pay.php

```diff
@@ -0,0 +1,472 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC_Stripe_Apple_Pay class.
+ *
+ * @extends WC_Gateway_Stripe
+ */
+class WC_Stripe_Apple_Pay extends WC_Gateway_Stripe {
```

Comment:
I haven't tested this but looks like this class can live without extending `WC_Gateway_Stripe`, or are there some methods we used from the parent that I missed?